extension/src/background/helpers/dataStorage.ts

```diff
@@ -151,8 +151,8 @@ const migrateTokenIdList = async () => {
       [NETWORKS.FUTURENET]: tokenIdsByKey,
     };
     await localStore.setItem(TOKEN_ID_LIST, newTokenList);
+    await migrateDataStorageVersion("1.0.0");
```

Comment:
oof - this was kind of a gotcha. Because this wasn't inside the conditional, on every update, the version was being set to `1.0.0`, which meant is going down to the whole list of migrations. Could this be why we've had a couple reports of data changing?

Reply:
I believe we should only be upping the migration version if a migration has been applied, but check my logic on this!

Reply:
ah man yeah that makes sense, looks like the last few were always incrementing it 😞 
I still don't see how this could cause the network switch. I see that `migrateSorobanRpcUrlNetwork` is the last network related migration to run but it only updates the selected network if it is futurenet so doesn't totally explain the reported behavior. This def should be only incrementing inside the version check though.